docker/docker-compose.yaml

```diff
@@ -195,6 +195,7 @@ services:
     volumes:
       # Mount the storage directory to the container, for storing user files.
       - ./volumes/app/storage:/app/api/storage
+      - ./volumes/app/chromadb:/app/chromadb
```

Comment:
Could you please explain this change?

Reply:
@laipz8200 Local deployment requires a vector database. Here I hope that the vdb data can be persisted, so it is mapped to the ./volumes/app/chromadb directory.

Reply:
I believe that the deployment of third-party services shouldn't be included in the dify configuration. If it needs to share a vdb with dify, you can apply this configuration in your local environment.

Reply:
hi @laipz8200 ,already deleted it in yaml, please help check，thank you！